src/storage/engine/mod.rs

```diff
@@ -55,15 +54,23 @@ pub trait Snapshot {
 
 #[derive(Debug, Clone, Copy)]
 pub enum Dsn<'a> {
-    Memory,
     RocksDBPath(&'a str),
     RaftKv,
 }
 
-// Now we only support Memory and RocksDB.
+pub fn temp_path() -> String {
+    TempDir::new("tests")
```

Comment:
I don't this will work as expect. Because when this function return, `drop` for `TempDir` will be invoked, so the temporary directory won't be deleted when the tests end.